modules/util/sass-compiler-jni/build.gradle

```diff
@@ -28,7 +31,7 @@ buildLibSass_Darwin {
 		buildShFile.write("""\
 LIBSASS_VERSION=\"${project.libSassVersion}\"
 autoreconf --force --install
-./configure --disable-tests --enable-shared
+./configure --disable-tests --disable-static --enable-shared
```

Comment:
By disabling static linking, is this going to make us run into compatibility issues with various linux distros?

Reply:
good point... testing on a different distro...

Reply:
ok i reread the makefile... and enabling the shared build is all that matters.  the static builds a library we don't use

Reply:
tested on ubuntu x64 also to make sure